### PR TITLE
Added parallel plane gravity

### PR DESCRIPTION
**Pull request by aisclark91**
## PR summary

I introduced a parallel plane gravity inside the function `Gravity::add_pointmass_to_gravity` and two option parameters in __cpp_parameters.

## PR checklist

- [ ] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

**Comment by zingale** [comment]
this fails to compile in the test suite:
http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2021-09-10-003/index.html

**Comment by aisclark91** [comment]
yes, there was an underscore missing in the _cpp_parameter file.

File: Source/gravity/Gravity.cpp [hunk 1]
```diff
@@ -2927,6 +2927,19 @@ Gravity::add_pointmass_to_gravity (int level, MultiFab& phi, MultiFab& grav_vect
             Real y = problo[1] + (static_cast<Real>(j) + 0.5_rt) * dx[1] - problem::center[1];
             Real z = problo[2] + (static_cast<Real>(k) + 0.5_rt) * dx[2] - problem::center[2];
 
+            if(castro::point_mass_offset_is_true == 1){
+                Real star_radius = castro::point_mass_location_offset;
+
+            if(AMREX_SPACEDIM == 1){
+                x += star_radius;
```
Comment (zingale) [inline Source/gravity/Gravity.cpp:2934]: can you fix this indenting?  right now it appears like this if block is not nested inside the one above.

**Comment by zingale** [comment]
great -- one last bit -- can you add a description of this to the CHANGES.md file?

**Comment by aisclark91** [comment]
changes were included in changes.md.

**Review by zingale — APPROVED** [review]

**Pull request merged**